docs/quick-start.md

```diff
@@ -10,7 +10,7 @@ To get started quickly, you can use the quick start manifest which will install
 
 ```sh
 kubectl create ns argo
-kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo/stable/manifests/quick-start-postgres.yaml
+kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo/stable/manifests/install.yaml
```

Comment:
I think this is probably the correct script for quick start as it installs MinIO and Postgres allowing a user to test out artifact storage and workflow archiving capabilities.

Instead, could you mention the alternative installation options?